recipes/modern-cpp-kafka/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class ModernCppKafkaConan(ConanFile):
+    name = "modern-cpp-kafka"
+    description = "A C++ API for Kafka clients (i.e. KafkaProducer, KafkaConsumer, AdminClient)"
+    license = "Apache License 2.0"
+    topics = ("kafka", "librdkafka", "kafkaproducer", "kafkaconsumer")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/morganstanley/modern-cpp-kafka"
+    settings = "arch", "build_type", "compiler", "os"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*", dst="include", src=os.path.join(self._source_subfolder, "include"))
```

Comment:
```suggestion
        self.copy(pattern="*.h", dst="include", src=os.path.join(self._source_subfolder, "include"))
```
filter only headers. There is a cmake file the middle